autoload/SpaceVim/layers/lang/julia.vim

```diff
@@ -9,32 +9,47 @@
 ""
 " @section lang#julia, layer-lang-julia
 " @parentsection layers
-" The layer provides synatax highlight and linting for julia. 
+" The layer provides synatax highlight julia. 
 " The completeion only works in nvim with deoplete.
 " However, the julia-vim could not be load on-demanding 
 " due to its LaTeXToUnicode feature.
 "
 
 function! SpaceVim#layers#lang#julia#plugins() abort
   let plugins = []
-  call add(plugins, ['JuliaEditorSupport/julia-vim' ])
-  call add(plugins, ['zyedidia/julialint.vim', {'on_ft' : 'julia'}])
+  call add(plugins, ['JuliaEditorSupport/julia-vim' , {'merged' : 0}])
   return plugins
 endfunction
 
 
 function! SpaceVim#layers#lang#julia#config() abort
+  " registe code runner commmand for julia
+  call SpaceVim#plugins#runner#reg_runner('julia', 'julia %s')
+  " registe REPL command and key bindings for julia
+  call SpaceVim#plugins#repl#reg('julia', 'julia')
+  let g:_spacevim_mappings_space.l.s = {'name' : '+Send'}
+  call SpaceVim#mapping#space#langSPC('nmap', ['l','s', 'i'],
```

Comment:
the `i` is a abbreviation of `interactive`?

Reply:
yeah, these repl command are same as other language layer. also in spacemacs.